Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -16153,7 +16153,7 @@
 			outputFileListPaths = (
 			);
 			outputPaths = (
-				"$(DSTROOT)$(LAUNCHD_PLIST_INSTALL_PATH)/$(LAUNCHD_PLIST_FILE_NAME)",
+				"$(DERIVED_FILE_DIR)/newOutputFile",
```

Comment:
I think this is supposed to be:
```suggestion
				"$(LAUNCHD_PLIST_OUTPUT_FILE)",
```

Reply:
Agreed.  I made that change, so I don't know why it didn't take.  I'll post an updated PR.

Reply:
r=me once this is updated!